hooks/tk-houdini_actions.py

```diff
@@ -116,4 +137,147 @@ def _merge(self, path, sg_publish_data):
         hou.hipFile.merge(path,
                           node_pattern="*",
                           overwrite_on_conflict=False,
-                          ignore_load_warnings=False)       
+                          ignore_load_warnings=False) 
+
+
+    ##############################################################################################################
+    def _import(self, path, sg_publish_data):
+        """Import the supplied path as a geo/alembic sop.
+        
+        :param str path: The path to the file to import.
+        :param dict sg_publish_data: The publish data for the supplied path.
+        
+        """
+
+        import hou
+        app = self.parent
+
+        name = sg_publish_data.get("name", "alembic")
+        path = self.get_publish_path(sg_publish_data)
+
+        obj_context = _get_current_context("/obj")
+        app.log_debug("Creating geo: %s" % (name,))
+
+        try:
+            geo_node = obj_context.createNode("geo", name)
+        except hou.OperationFailed:
+            # failed to create the node in this context, create at top-level
+            obj_context = hou.node("/obj")
+            geo_node = obj_context.createNode("geo", name)
+
+        # delete the default nodes created in the geo
+        for child in geo_node.children():
+            child.destroy()
+
+        app.log_debug("Creating alembic sop: %s" % (name,))
+        alembic_sop = geo_node.createNode("alembic", name)
+        app.log_debug("Setting alembic sop path: %s" % (path,))
```

Comment:
Could these debug statements be consolidated so there wouldn't be so many lines?  Not a big deal, but it seems a little redundant.